docs/technical_references/design_docs/nordic_eits.rst

```diff
@@ -0,0 +1,73 @@
+====================================================
+Encryption in Internal Trusted Storage (ITS) Service
+====================================================
+
+:Author: TBD
+:Organization: TBD
+:Contact: TBD
+
+Introduction
```

Comment:
The introduction seems more argumentative than introductive. Would it make sense to have something call Justification, and leave the introduction to simply introduce the feature?